org.freedesktop.Sdk.Extension.node12.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: org.freedesktop.Sdk.Extension.node12
+branch: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /usr/lib/sdk/node12
+  prepend-path: /usr/lib/sdk/node12/bin
+  prepend-ld-library-path: /usr/lib/sdk/node12/lib
+cleanup:
+  - '/bin/pip*'
+  - '/bin/python*config'
+  - '/bin/smtpd.py'
+  - '/lib/pkgconfig'
+  - '/share/doc'
+  - '/share/man'
+  - '/share/systemtap'
+modules:
+  - shared-modules/python2.7/python-2.7.json
```

Comment:
Does Node.js 12 still require Python 2?

Reply:
For building, yes -- use of python 3 is still experimental (according to the readme)